go.mod

```diff
@@ -62,7 +62,8 @@ require (
 	github.com/mitchellh/go-homedir v1.1.0
 	github.com/moby/buildkit v0.11.6
 	github.com/onsi/gomega v1.27.7
-	github.com/openshift/api v0.0.0-20200713203337-b2494ecb17dd
+	github.com/openshift/api v0.0.0-20230120195050-6ba31fa438f2
+	github.com/openshift/library-go v0.0.0-20230120214501-9bc305884fcb
```

Comment:
I would rather not bring in this large and dependency-heavy dependency just to bring in ~20 lines of code. Can we just copy them over?

Reply:
done